modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -67,6 +69,15 @@ const addArguments = (capabilities: Record<string, any>, name: string, args: str
   capabilities[name].args = currentArgs.concat(args);
 };
 
+const getExtraBrowserCapabilities = (settings: DriverSettings): string[] => {
+  const extraCaps = settings.extraBrowserCapabilities.trim();
+  if (extraCaps.length > 0) {
```

Comment:
No ternary I'm surprised. :wink:

Reply:
I thought it wasn't commonly used. I don't know what style to use any more. I'm just trying to fit in :)

Reply:
I was also surprised by this tbh 😂  I don't think we have a standard much anymore for this 😞